lib/jsdom/web-idl/implementation/EventTarget.js

```diff
@@ -0,0 +1,231 @@
+"use strict";
+const DOMException = require("../DOMException");
+const reportException = require("../../living/helpers/runtime-script-errors");
+const domSymbolTree = require("../../living/helpers/internal-constants").domSymbolTree;
+
+class EventTarget {
+  addEventListener(type, callback, capture) {
+    if (callback === undefined || callback === null) {
```

Comment:
All of this (lines 8 through 16) should be handled by webidl2js, no? It's OK if it isn't right now, but we should at least add comment delimiters.